tests/test_script.py

```diff
@@ -7,7 +7,7 @@
 
 
 def test_module_with_explicit_whitelists():
-    assert call_vulture(["vulture/"] + WHITELISTS) == 0
+    assert call_vulture(["vulture/"] + list(WHITELISTS)) == 0
```

Comment:
If WHITELISTS is a generator now, we should turn it into a list when it is created. Otherwise, we can drop the list() call here.